person.rb

```diff
@@ -0,0 +1,26 @@
+class Person
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  attr_accessor :name, :age
+
+  def id
+    @age
+  end
+
+  private
+
+  def of_age?
+    @age > 18
+  end
+
+  public
+
+  def can_use_services?
+    (of_age? || parent_permission?)
+  end
```

Comment:
- [ ] Kindly use this function above `private`, then you don't have to use the keyword `public`.